services/user.py

```diff
@@ -0,0 +1,47 @@
+from django.db.models import QuerySet
+from db.models import User
+
+
+def create_user(username: str,
+                password: str,
+                email: str | None = None,
+                first_name: str | None = None,
+                last_name: str | None = None) -> QuerySet[User]:
+    user = User.objects.create_user(username=username, password=password)
+
+    if email:
+        user.email = email
+    if first_name:
+        user.first_name = first_name
+    if last_name:
+        user.last_name = last_name
+
+    user.save()
+    return user
+
+
+def get_user(user_id: int) -> QuerySet[User]:
+    user = User.objects.get(pk=user_id)
```

Comment:
it is better to use get_user_model() method instead of User
fix this everywhere